_posts/2022-10-18-trino-summit-2022-teaser-2.md

```diff
@@ -0,0 +1,141 @@
+
+I urge you to join us either in-person or virtually if you are a fan of Trino,
+big data, open source, data engineering, Java, or all the above! This conference
+is free and takes place on November 10th. Check out the details below!
+
+<!--more-->
+
+## Register for the summit
+
+Trino Summit is being held at the Commonwealth Club in San Francisco, CA.
+Make sure you register quickly for in-person registration, as it is limited to
+250 seats! Virtual registration is also picking up quickly and is limited to
+1500 registrants. Register today before spots run out!
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-pink" href="https://www.starburst.io/info/trinosummit/">
+        Register now
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+### Trino Summit 2022 teaser
+
+Check out the teaser for this year's Trino SUmmit and get ready to Federate 'em
+all!
+
+<iframe src="https://www.youtube.com/embed/o2MJvRKG14M" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px;
+margin-bottom:5px; max-width: 100%;" allowfullscreen="">
+</iframe>
+
+
+## Announcing the second round of sessions and speakers
+
+As we announced in the [previous summit teaser]({% post_url
+2022-09-22-trino-summit-2022-teaser %}), we have already announced some of our
+exciting lineup of speakers! The topics range from architectures like data mesh
+and data lakehouse, to running Trino at scale with fault-tolerant execution, and
+of course, query federation. 
+
+We have a full roster planned but here is a glance at a few full confirmed
+sessions. Stay tuned for future blog posts as we announce more session as they
+are confirmed!
+
+### SK Telecom's journey to Iceberg
+
+SK Group is one South Korea's largest conglomerates in the nation covering
+industries from manufacturing to telecommunications. SK Telecom uses an
+on-premise data platform at petabyte scale using Trino as a query engine. We
+chose Trino for its ability to connect to heterogeneous data sources and ensures
+fast performance that plays a key role in our data platform.
+
+As data along with user demands to analyze long-term data increased, the Trino
+Hive connector faced several challenges. When processing a heavy query whose
+input data size exceeded a terabyte, it puts a great burden on the cluster,
+causing some jobs to fail. This can be problematic as Trino's resource sharing
+architecture can affect multiple users when a heavy query occurs.
+
+To address this situation, we optimized the data structure, tuned query, and
+used the resource group to separate them, but none of this fixed the problem. We
+learned about the advantages to using Apache Iceberg, and realized this could
+address some of these scaling issues we were facing. In this talk, we will share
+our journey.
+
+JaeChang Song, Data Engineer at SKTelecom and Trino/Iceberg Contributor
```

Comment:
Maybe use a bullet list for presenters and use italics ?

Reply:
I like this, done and applied to the original teaser.